### PR TITLE
Cawatson verify webhook

### PR DESCRIPTION
**Pull request by dstarr**
David modified the code in casey's PR to allow for dev env wherein webhook does not check the operation details.

### CODE REVIEW

**Review by code4clouds — CHANGES_REQUESTED** [review]
This needs an update in the readme file on how to setup the debug mode.

**Comment by dstarr** [comment]
Added instructions for using `environment=development` in production.

**Review by code4clouds — APPROVED** [review]

File: src/SaaS.SDK.CustomerProvisioning/appsettings.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
   //  "signedoutredirecturi": "",
   //  "fulfillmentapiversion": "2018-08-31",
   //  "adauthenticationendpoint": "https://login.microsoftonline.com",
-  //  "saasappurl": ""
+  //  "saasappurl": "",
+  //  "environment: ""
```
Comment (santhoshbomma9) [inline src/SaaS.SDK.CustomerProvisioning/appsettings.json:23]: @dstarr typo!

Comment (santhoshb-msft) [reply src/SaaS.SDK.CustomerProvisioning/appsettings.json:23]: fixed typo #312

**Pull request merged**